plutus-core-spec/plutus-core-specification.tex

```diff
@@ -595,7 +591,7 @@ \section{Type Correctness}
 %
 %     \begin{prooftree}
 %        \AxiomC{\(\ctxni{\Gamma}{\termJ{x}{A}}\)}
-%        \AxiomC{\(x \not= y\)}        
+%        \AxiomC{\(x \not= y\)}
 %        \BinaryInfC{\(\ctxni{\Gamma, \termJ{y}{B}}{\termJ{x}{A}}\)}
 %    \end{prooftree}
```

Comment:
Since this has appeared in the diff - can we just delete all commented-out stuff?